bin/hdfcoinc/pycbc_multiifo_combine_statmap

```diff
@@ -26,11 +26,13 @@ f = h5py.File(args.output_file, "w")
 logging.info('Copying segments and attributes to %s' % args.output_file)
 # Move segments information into the final file - remove some duplication in earlier files
 for fi in files:
-    for key in f['segments']:
-        f['segments/%s/end' % key] = fi['segments/%s/end' % key][:]
-        f['segments/%s/start' % key] = fi['segments/%s/start' % key][:]
+    for key in fi['segments']:
         if key.startswith('foreground') or key.startswith('background'):
```

Comment:
the `startswith(foreground)` check needs to be removed, as we're now going to check that later

Reply:
we do need the test, because we do not want to create a f['segments/foreground_veto/end'] key. That was the first error I corrected. Each file would try to overwrite the same key and the job fails. that is why this test is there.

Reply:
Yes, sorry I'd missed the later %s